src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsLambda/HandlerMethodWrapper.cs

```diff
@@ -155,6 +158,11 @@ private void InitLambdaData(InstrumentedMethodCall instrumentedMethodCall, IAgen
                     {
                         agent.Logger.Log(Agent.Extensions.Logging.Level.Debug, $"Supported Event Type found: {_functionDetails.EventType}");
                     }
+                    else if (!_unsupportedInputTypes.Contains(name))
```

Comment:
Just FYI, `Add` returns false if the item is already present, so you could simplify this with `if (_unsupportedInputTypes.Add(name))`  (and similarly below with `_unexpectedResponseTypes`)